docs/roadmap.md

```diff
@@ -2,10 +2,15 @@
 
 These are our upcoming features:
 
-- Support extracting runtime configurations from additional Cloud providers such as Azure & GCP
-- CIS certifications for k8s and release 1.5
-- Integration with AWS Security Hub
-- Extend coverage of queries for GCP & Azure
+- Release of the v1.6
+- KICS will support the following technologies:
+    - Pulumi
+    - Serverless Framework
+    - Crossplane
+    - Knative
+- Kubernetes API
```

Comment:
Maybe refer to Kubernetes API as kuberneter to reduce inconsistency between documentation
[docs/integrations_kuberneter.md](https://github.com/Checkmarx/kics/pull/5651/files#diff-d0564607bb5111b6ff222af74f8fc22b08d3537bb23c1878fad9562a62bcafb4)

Reply:
which is the best approach in your opinion:
1) Kubernetes API ( Kuberneter )
2) Kuberneter Integration